develop/models/datamodel.js

```diff
@@ -0,0 +1,36 @@
+var DataModel = function (getJSON, map) {   // used dependency injection DataModel($.getJSON, $.map)
```

Comment:
Please look on knockoutjs tutorial how to create DataModels (http://learn.knockoutjs.com/)